### ISSUE TITLE
Are you still trying to get the painter working?

### ISSUE THREAD

**Issue by aleccaruana** [IMRZ/twtk-interactive-map#16]
Title says it all. That would make this app a lot more functional.

**Comment by IMRZ** [comment]
Hey @aleccaruana, I was bit occupied with other projects. I will get something basic up and running this week. Thanks!

**Comment by IMRZ** [comment]
@aleccaruana I have a basic map painter up and running. I will add in settlement markers soon.

I am not so sure about letting user's place markers or images, maybe as a separate map/mode. I will try and see if I can get it to work.

**Comment by aleccaruana** [comment]
Looks great so far! My only other suggestion would be that it would be better if the faction names on the painter were just that of the leader at a given start date rather than the kingdom name ('Wei,' 'Zhang,' etc.).

**Comment by aleccaruana** [comment]
The map does look better without the resource-based settlement markers. Maybe you could put the resource in each county (horses, lumber yard, etc.) in the tooltip when you hover over each county?

**Comment by IMRZ** [comment]
Yeah, I'm still trying out some different things and see what works. I've added some extra text in the tooltip and I have also added a way to switch between marker types: flags, resources and simple settlements.
